appcues/src/main/java/com/appcues/trait/appcues/CarouselTrait.kt

```diff
@@ -1,7 +1,11 @@
 package com.appcues.trait.appcues
 
+import androidx.compose.foundation.ExperimentalFoundationApi
 import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.BoxScope
+import androidx.compose.foundation.pager.HorizontalPager
+import androidx.compose.foundation.pager.PagerState
+import androidx.compose.foundation.pager.rememberPagerState
```

Comment:
the changes in this file are to use the new HorizontalPager and related items from Compose, instead of Accompanist